src/toolkits/util/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ make_library(unity_util OBJECT
     sframe_utils.cpp
     class_registrations.cpp
     random_sframe_generation.cpp
+    float_array_serialization.cpp
```

Comment:
is there any coding standard around keeping cmakelists sorted alphabetically?

Reply:
We have been using https://google.github.io/styleguide/cppguide.html in general, but it doesn't explicitly address CMakeLists. Extrapolating from its treatment of headers, I would agree that we should sort these....